Training6/thongnq-training6-task2/src/components/NetworkStatus/index.js

```diff
@@ -0,0 +1,21 @@
+
+import React from 'react';
+import { Para } from './NetworkStatus.styles';
+
+import {useDispatch, useSelector} from "react-redux";
+
+import {updateNetwork} from "../../state/actions/network"
+
+const NetworkStatus = () => {
+    const dispatch = useDispatch();
+
+    const status = useSelector(state => state.network.status);
+
+    const handleChangeStatus = () => {
+        dispatch(updateNetwork());
+    }
+
+    return <Para>Network: <button onClick={handleChangeStatus}>{status}</button></Para>
```

Comment:
@Thong-Nguyen-Zigvy network status should get direct from web API, and as suggestion on the task, we should move it to saga even channel